### PR TITLE
[MAR-2475] seller dashboard

### PR DESCRIPTION
**Pull request by malmas18**

### CODE REVIEW

File: app/main/views/login.py [hunk 1]
```diff
@@ -137,9 +138,14 @@ def send_invite_user():
 
         try:
             send_email(
-                form.email_address.data,
+                email_address,
                 email_body,
-                current_app.config['INVITE_EMAIL_SUBJECT'],
+                'Invitation to join {} as team member'.format(
```
Comment (tobiasbrown) [inline app/main/views/login.py]:
```suggestion
                'Invitation to join {} as a team member'.format(
```

File: app/templates/emails/invite_user_email.html [hunk 2]
```diff
@@ -2,24 +2,22 @@
 <html>
 <head lang="en">
     <meta charset="UTF-8">
-    <title>Your Digital Marketplace invitation</title>
+    <title>Invitation to join {{ supplier }} as team member</title>
```
Comment (tobiasbrown) [inline app/templates/emails/invite_user_email.html]:
```suggestion
    <title>Invitation to join {{ supplier }} as a team member</title>
```

File: app/templates/emails/invite_user_email.html [hunk 3]
```diff
@@ -2,24 +2,22 @@
...
-	<p>{{ user }} has invited you to become a contributor for {{ supplier }} on the Digital Marketplace.</p>
+	<p>{{ user }} has invited you to join {{ supplier }} as a team member on the Digital Marketplace.</p>
 
-	<p>Please <a href="{{ url }}">click this link</a> to accept the invitation</p>
+	<p>To accept, you must <a href="{{ url }}">register for a Digital Marketplace account</a> within 7 days.</p>
 
-	<p>You must click the above link within 7 days of receiving this email.</p>
-
-	<p>This is an automated message. Email <a href="mailto:marketplace@digital.gov.au">marketplace@digital.gov.au</a> if you need help.</p>
+	<p>If you have any questions or need guidance, please <a href="mailto:marketplace@digital.gov.au">contact the Marketplace</a>.</p>
```
Comment (tobiasbrown) [inline app/templates/emails/invite_user_email.html]: In the `create_user_error` template, you use `generic_contact_email` to pull the email address from config.  Should we make it consistent?

File: tests/app/main/test_login.py [hunk 4]
```diff
@@ -168,12 +166,12 @@ def test_should_call_send_invitation_email_with_correct_params(self, send_email,
...
             )
 
-            assert res.status_code == 302
+            assert res.status_code == 200
 
             send_email.assert_called_once_with(
                 "email@email.com",
                 mock.ANY,
-                "SUBJECT",
+                'Invitation to join Supplier Name as team member',
```
Comment (tobiasbrown) [inline tests/app/main/test_login.py]:
```suggestion
                'Invitation to join Supplier Name as a team member',
```

File: app/templates/emails/invite_user_email.html (hunk 3 above)
Comment (malmas18) [reply app/templates/emails/invite_user_email.html]: updated

**Pull request merged**